x-pack/plugins/security/common/licensing/license_service.test.ts

```diff
@@ -43,15 +47,33 @@ describe('license features', function() {
     });
   });
 
+  it('should notify consumers of licensed feature changes', () => {
+    const rawLicense$ = new BehaviorSubject(getMockRawLicense({ isAvailable: false }));
+    const serviceSetup = new SecurityLicenseService().setup({
+      license$: rawLicense$,
+    });
+
+    const subscriptionHandler = jest.fn();
+    const subscription = serviceSetup.license.features$.subscribe(subscriptionHandler);
+    try {
+      expect(subscriptionHandler).toHaveBeenCalledTimes(1);
```

Comment:
nit: should we make sure that we're actually getting the features emitted by `features$`

Reply:
Yep, I didn't want to flesh out this test suite until we had settled on the approach. I'll update once we do so.